apps/meteor/client/views/admin/subscription/components/cards/AppsUsageCard/AppsUsageCard.spec.tsx

```diff
@@ -36,9 +36,7 @@ it('should render data as progress bars', async () => {
 
 	await userEvent.click(screen.getByRole('button', { name: 'Click_here_for_more_info' }));
 
-	expect(
-		screen.getByText('Community workspaces can enable up to 5 marketplace apps. Private apps can only be enabled in premium plans.'),
-	).toBeInTheDocument();
+	expect(screen.getByRole('link', { name: 'premium plans' })).toHaveAttribute('href', 'https://go.rocket.chat/i/pricing-product');
```

Comment:
I recommend using the same constant `PRICING_LINK` here. In case this link changes in the future the test will change with it. _(Consider this for all instances in this file)_